src/components/Api.js

```diff
@@ -1,30 +1,26 @@
 export class Api {
```

Comment:
It’s better to move api class to utils directory, since it's not a component, but a utility class

Reply:
I moved the api class to the utils directory.